services/repository.py

```diff
@@ -253,9 +253,11 @@ async def update_commit_from_provider_info(repository_service, commit):
             # so we append the branch name with the fork slug
             branch_name = commit_updates["head"]["branch"]
             # TODO: 'slug' is in a `.get` because currently only GitHub returns that info
-            if commit_updates["head"].get("slug") != commit_updates["base"].get("slug"):
-                branch_name = commit_updates["head"]["slug"] + ":" + branch_name
-            commit.branch = branch_name
+            # if this PR is from a fork don't update the branch here
```

Comment:
Why is this the case? Can you maybe share some examples, please?

Reply:
updated the PR to make more sense, basically with the old behaviour this would update the branch name to be:
`<username>/<repo name>:<branch name>` with this change it will be `<username>:<branch name>`